test/unit/st-client.test.ts

```diff
@@ -0,0 +1,51 @@
+import axios from '../../__mocks__/axios'
+import {
+	BearerTokenAuthenticator,
+	SmartThingsClient,
+} from '../../src'
+
+
+describe('ST Client', () => {
+	afterEach(() => {
+		axios.request.mockReset()
+	})
+
+	it('Construction with no location', async () => {
```

Comment:
The test names here can be `test` if not using sentence-like test names. https://stackoverflow.com/a/47399770